chroma-manager/tests/integration/utils/test_blockdevices/test_blockdevice_linux.py

```diff
@@ -21,6 +21,10 @@ def device_path(self):
         return self._device_path
 
     @property
+    def wipe_signatures_commands(self):
+        return ['wipefs -a %s' % self.device_path]
```

Comment:
Isn't `wipefs` a generic tool and not specific to ldiskfs (i.e. ext4)?

If so, shouldn't this be hoisted to the parent class?